websites/A/AniWorld/presence.ts

```diff
@@ -51,55 +51,64 @@ presence.on(
 presence.on("UpdateData", async () => {
 	const page = document.location.pathname,
 		presenceData: PresenceData = {
-			largeImageKey: "anicloud",
+			largeImageKey: "aniworld-logo",
 			startTimestamp: Math.floor(Date.now() / 1000)
 		};
 	if (page === "/") presenceData.details = "Betrachtet die Startseite";
 	else if (page.startsWith("/anime/")) {
-		presenceData.details = `${
-			(document.querySelector("h1") as HTMLElement).textContent
-		}`;
-		presenceData.state = `${(
-			document.querySelector("title") as HTMLElement
-		).textContent
-			.split("Staffel")[0]
-			.replace("Filme von", " ")
-			.split(" | AniCloud.io - Animes gratis online ansehen")} - ${
-			(document.querySelector("h2") as HTMLElement).textContent
-		}`;
-		[presenceData.startTimestamp, presenceData.endTimestamp] =
-			presence.getTimestamps(currentTime, timeEnd);
-		presenceData.buttons = [
-			{
-				label: "Watch Anime",
-				url: document.location.href
-			}
-		];
+		// Check if we are on the episode selection page
+		if (page.split("/").length === 4) {
+			presenceData.details = `${
+				(document.querySelector("h1") as HTMLElement).textContent
```

Comment:
Use Type Generic and don't use HTMLElement, use the proper typing for it

Reply:
Hey there, this is code I moved from below, which I didnt really change.
Am I interpreting you right that I should remove the "as HTMLElement" part of Line 62?

Reply:
Use the proper type if you are going to use a type and use type generics so `.querySelector<HTMLHeadingElement>` in the case of L62